src/guides/v2.3/frontend-dev-guide/css-topics/using-fonts.md

```diff
@@ -47,7 +47,7 @@ To ensure the stability of your customizations and prevent upgrades from overwri
    }
    ```
 
-   `@font-format` is optional. If you need to specify a format, enclose it in single quotes. For example, `@font-format: 'ttf'` for TrueType Fonts. The available types are: "woff", "woff2", "ttf", "oet", "otf", and "svg"
```

Comment:
Should this not also be corrected in - src/guides/v2.3/frontend-dev-guide/css-topics/using-fonts.md